.github/workflows/docker_image.yml

```diff
@@ -1,38 +1,47 @@
-name: Docker Package
+name: Publish Docker Packages
 on:
   workflow_dispatch:
   push:
-    tags: "*"
     branches:
       - main
   pull_request:
 
-env:
-  REGISTRY: ghcr.io
-  IMAGE_NAME: Kitware/pan3d-viewer # ${{ github.repository }}
 
 jobs:
-  build-and-publish:
+  geotrame-viewer:
     runs-on: ubuntu-latest
-    if: github.event_name == 'push' || github.event_name == 'workflow_dispatch' || ( github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository )
     steps:
       - uses: actions/checkout@v3
-      - name: Log into the Container registry
-        uses: docker/login-action@v2
         with:
-          registry: ${{ env.REGISTRY }}
-          username: token
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Extract metadata for the Docker image
-        id: meta
-        uses: docker/metadata-action@v4
+          fetch-depth: 0
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
         with:
-          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
       - name: Build and push the Docker image
         uses: docker/build-push-action@v3
         with:
           context: .
           file: docker/Dockerfile
           push: ${{ github.actor != 'dependabot[bot]' }}
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          tags: kitware/geotrame-viewer
```

Comment:
We don't have a dockerhub repository here... It needs to be `kitware/trame:geotrame-viewer`

Reply:
Changed in [3e3251d](https://github.com/Kitware/geo-trame/pull/77/commits/3e3251d2c0b1196a2cfbd2a34d9871051439bbea)